content/docs/command-reference/run.md

```diff
@@ -6,13 +6,16 @@ command.
 ## Synopsis
 
 ```usage
-usage: dvc run [-h] [-q | -v] [-n <name>] [--no-exec] [--no-run-cache]
-               [--no-commit] [-d <path>] [-o <filename>] [-O <filename>]
-               [-p [<filename>:]<params_list>] [-m <path>] [-M <path>]
-               [--plots <path>] [--plots-no-cache <path>] [--live <path>]
-               [--live-no-cache <path>] [--live-no-html]
-               [-w <path>] [-f] [--outs-persist <filename>]
-               [--outs-persist-no-cache <filename>] [-c <filename>]
+usage: dvc run [-h] [-q | -v] [-n <name>] [-f]
+               [--no-exec] [--no-run-cache] [--no-commit]
```

Comment:
They just looked like they can be grouped 😋 . Maybe they should be closer to the bottom of the list since they're probably not used very often?

Reply:
Agreed. I'd probably put them at the bottom not only because they're not used often but also because it makes the options list look similar to `dvc stage add`..